---
layout: about
title: about
permalink: /
subtitle:
cv_pdf: example_pdf.pdf
profile:
  align: right
  image: prof_pic.jpg
  image_circular: false # crops the image to make it circular
  address: >
    <p>1 Baylor Plaza, Houston,</p>
    <p>TX 77030, USA</p>

news: false  # includes a list of news items
latest_posts: false  # includes a list of the newest posts
selected_papers: true # includes a list of papers marked as "selected={true}"
social: true  # includes social icons at the bottom of the page
---
Hi! I am a PhD student at the [Genetics and Genomic program](https://www.bcm.edu/education/graduate-school-of-biomedical-sciences/programs/genetics-genomics) in [Baylor college of mdeidince](https://www.bcm.edu/).

My research interests include cancer epigenomics and machine learning with a focus on developing models/tools for gaining biological insights from omics data that facilitates cancer diagnosis and treatments.

In 2021, I joined the [Institute of Information Science, Acadmeia Sinica](https://www.iis.sinica.edu.tw/zh/index.html) and developed a deep learning model for predicting extrachromosomal circular DNA, which was advised by [Huai-Kuang Tsai](https://homepage.iis.sinica.edu.tw/pages/hktsai/index_en.html). Previously, I did a remote internship at German Cancer Research Center (DKFZ) advised by [Pavlo Lutsik](https://www.dkfz.de/en/CanEpi/staff/Computational_epigenomics.html). During this internship, I developed a R package along with a web user interface for analysis of epignomics data from The Cancer Genome Atlas (TCGA). Before that, in my master's thesis, which was advised by [Shaw-Jenq Tsai](https://www.binfo.ncku.edu.tw/SJT_Lab_Web/index.html), I analyzed transcriptomic and epigenomic data to elucidate the role of histone modifications in colon cancer drug resistance. My undergraduate project advised by [Kuei-Yang Hsiao](https://kyhsiao.wixsite.com/lab-x-omics/pi) involves developing bioinformatics tools for identification and functional characterization of circular RNAs.
